src/main/java/org/vanilladb/core/storage/file/FileMgr.java

```diff
@@ -184,7 +186,7 @@ void write(BlockId blk, IoBuffer buffer) {
 	}
 
 	/**
-	 * Appends the contents of a byte buffer to the end of the specified file.
+	 * Append will not write to file, call write to make to write buffer to file.

```

Comment:
This document should first describe `Gets the block id of the new block appended in the specified file.`, then the warning message.